### PR TITLE
Add CSS Ball 2023 page

### PR DESCRIPTION
**Pull request by LMBishop**
👀

### CODE REVIEW

File: _data/ball/2023.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  - question: What is a Ball?
+    answer: A ball is a solid or hollow spherical or egg-shaped object that is kicked, thrown, or hit in a game. <br><br> Ok but seriously, a Ball is an opportunity to come together and celebrate the year. Many societies hold them for their members and we're no different. Our annual Ball is the biggest night of the year so if you're not attending you're really missing out.
+  - question: What will be there?
+    answer: The ball will have a 3 course meal, a bar, and a dancefloor. We will also have a live band, a photobooth, and a photographer.
+  - question: When can I buy tickets?
+    answer: Now! You'll need to be a CSS Member to purchase tickets - if you aren't then you can join CSS on the <a href="/join">Guild website</a>.
+  - question: How much do tickets cost?
+    answer: Tickets are £42.50, same as last years. The Guild will also add on a booking fee - we can't do anything about that.
+  - question: Can I bring a guest?
+    answer: Yes, you will need to buy an extra ticket for your guest.
```
Comment (JTWWilson) [inline _data/ball/2023.yaml:44]: Can we write a slightly passive aggressive line in here saying that the Guild haven't enabled this on the website at the moment but we will be fixing this as soon as we can?

**Review by JTWWilson — COMMENTED** [review]
Good to go except misleading bit about the Guild ticket site 😑

File: _data/ball/2023.yaml (hunk 1 above)
Comment (JTWWilson) [reply _data/ball/2023.yaml]: oh honey I have _so_ much more where that came from. Enjoy this game of 'spot the difference' between what we requested and what the Guild put up originally.
cursed_ball_tickets

Comment (codemicro) [reply _data/ball/2023.yaml]: 🤡

**Review by JTWWilson — APPROVED** [review]
Perfect wording. Approve

File: _data/ball/2023.yaml (hunk 1 above)
Comment (JTWWilson) [reply _data/ball/2023.yaml]: Oh another one @codemicro - there isn't enough room to fit 3 songs into the song request box: 
image

**Pull request merged**